src/eom-scroll-view.c

```diff
@@ -654,7 +654,7 @@ adjustment_changed_cb (GtkAdjustment *adj, gpointer data)
 	scroll_to (view, gtk_adjustment_get_value (priv->hadj),
 		   gtk_adjustment_get_value (priv->vadj), FALSE);
 
-	gtk_widget_queue_resize (view);
+	gtk_widget_queue_resize (priv->display);
```

Comment:
This is *not* the same as before: `priv->display` is the inner `GtkDrawingArea`, whereas `view` is the `EmoScrollView` itself.

To not change behavior, either add the cast `GTK_WIDGET(view)`, or use `data` directly (which is the untyped value cast to make `view`, which would not produce any warning either)

Reply:
done